endpoints/api/superuser_models_pre_oci.py

```diff
@@ -22,10 +22,20 @@
 from util.request import get_request_ip
 
 
+def _get_namespace_quotas(namespace_user):
+    if not features.QUOTA_MANAGEMENT:
```

Comment:
Nit: Feels weird could be just a more standard switch format with return under the true statement instead of a break style.